VolunteerApp/Data/VolunteerContext.cs

```diff
@@ -0,0 +1,51 @@
+﻿using VolunteerApp.Models;
+using Microsoft.EntityFrameworkCore;
+using System.Data;
+using System.Data.SqlClient;
+
+namespace VolunteerApp.Data
+{
+    public class VolunteerContext : DbContext
+    {
+        private IDbConnection DbConnection { get; }
+
+        private readonly IConfiguration _configuration;
+        public VolunteerContext (DbContextOptions<VolunteerContext> options , IConfiguration configuration)
+            : base(options)
+        {
+            this._configuration = configuration;
+            DbConnection = new SqlConnection(this._configuration.GetConnectionString("DefaultConnection"));
+            //Database.EnsureCreated();
```

Comment:
Please remove

Code is always better than comments. So always avoid comments where code can be made self explanatory.